### PR TITLE
chore: component ownership audit

### PR DESCRIPTION
**Pull request by dyladan**
- Adding all packages regardless of their ownership to the ownership yml
- Sort ownership yml alphabetically
- Add comment making it clear which packages are looking for ownership

### CODE REVIEW

**Review by MSNev — APPROVED** [review]

File: .github/component_owners.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ components:
     - carolabadeer
   detectors/node/opentelemetry-resource-detector-container:
     - abhee11
+  detectors/node/opentelemetry-resource-detector-gcp: []
```
Comment (dyladan) [inline .github/component_owners.yml:9]: Maybe someone from google would want to take this? @aabmass maybe?

File: .github/component_owners.yml [hunk 2]
```diff
@@ -6,6 +6,10 @@ components:
...
+    # Unmaintained?
+  detectors/node/opentelemetry-resource-detector-github: []
```
Comment (dyladan) [inline .github/component_owners.yml:11]: The original contributor here was @smithclay. Maybe they would like to take ownership?

File: .github/component_owners.yml [hunk 3]
```diff
@@ -17,67 +21,101 @@ components:
     - legendecas
   packages/opentelemetry-id-generator-aws-xray:
     - carolabadeer
+  packages/opentelemetry-propagation-utils: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:25]: Think @open-telemetry/javascript-maintainers or some subset should probably take this

```suggestion
    - dyladan
    - pichlermarc
    - legendecas
    - blumamir
```

File: .github/component_owners.yml [hunk 4]
```diff
@@ -17,67 +21,101 @@ components:
...
   packages/opentelemetry-redis-common:
     - haddasbronfman
+  packages/opentelemetry-test-utils: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:29]: Think @open-telemetry/javascript-maintainers or some subset should probably take this

```suggestion
    - dyladan
    - pichlermarc
    - legendecas
    - blumamir
```

File: .github/component_owners.yml (hunk 3 above)
Comment (dyladan) [inline .github/component_owners.yml:25]: Think @open-telemetry/javascript-maintainers or some subset should probably take this

```suggestion
  packages/opentelemetry-propagation-utils:
    - dyladan
    - pichlermarc
    - legendecas
    - blumamir
```

File: .github/component_owners.yml [hunk 5]
```diff
@@ -17,67 +21,101 @@ components:
...
+    - rauno56
   plugins/node/instrumentation-tedious:
     - rauno56
   plugins/node/opentelemetry-instrumentation-aws-lambda:
     - carolabadeer
   plugins/node/opentelemetry-instrumentation-aws-sdk:
     - carolabadeer
     - blumamir
+  plugins/node/opentelemetry-instrumentation-bunyan: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:51]: Originally contributed by @seemk 

Would you care to take ownership?

```suggestion
  plugins/node/opentelemetry-instrumentation-bunyan:
    - seemk
```

File: .github/component_owners.yml [hunk 6]
```diff
@@ -17,67 +21,101 @@ components:
...
     - rauno56
   plugins/node/opentelemetry-instrumentation-router:
     - rauno56
-  plugins/node/opentelemetry-instrumentation-pg:
-    - rauno56
-  plugins/node/opentelemetry-instrumentation-socket.io:
-    - mottibec
-    - rauno56
+  plugins/node/opentelemetry-instrumentation-winston: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:100]: Originally contributed by @seemk 

Would you care to take ownership?

```suggestion
  plugins/node/opentelemetry-instrumentation-winston:
    - seemk
```

File: .github/component_owners.yml [hunk 7]
```diff
@@ -17,67 +21,101 @@ components:
...
   plugins/node/opentelemetry-instrumentation-nestjs-core:
     - rauno56
-  plugins/node/opentelemetry-instrumentation-redis:
-    - blumamir
+  plugins/node/opentelemetry-instrumentation-net: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-pg:
+    - rauno56
+  plugins/node/opentelemetry-instrumentation-pino: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:90]: Originally contributed by @seemk 

Would you care to take ownership?

```suggestion
  plugins/node/opentelemetry-instrumentation-pino:
    - seemk
```

File: .github/component_owners.yml [hunk 8]
```diff
@@ -17,67 +21,101 @@ components:
...
     - rauno56
   plugins/node/opentelemetry-instrumentation-aws-lambda:
     - carolabadeer
   plugins/node/opentelemetry-instrumentation-aws-sdk:
     - carolabadeer
     - blumamir
+  plugins/node/opentelemetry-instrumentation-bunyan: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-cassandra: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:53]: Originally contributed by @seemk 

Would you care to take ownership?

```suggestion
  plugins/node/opentelemetry-instrumentation-cassandra:
    - seemk
```

File: .github/component_owners.yml [hunk 9]
```diff
@@ -17,67 +21,101 @@ components:
...
+  plugins/node/opentelemetry-instrumentation-connect: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-dns: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:57]: Originally contributed by @Flarna  

Would you care to take ownership?

```suggestion
  plugins/node/opentelemetry-instrumentation-dns:
    - Flarna
```

File: .github/component_owners.yml [hunk 10]
```diff
@@ -17,67 +21,101 @@ components:
...
+  plugins/node/opentelemetry-instrumentation-express: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:59]: Originally contributed by @vmarchaud  

Would you care to take ownership?

```suggestion
  plugins/node/opentelemetry-instrumentation-express:
    - vmarchaud
```

File: .github/component_owners.yml [hunk 11]
```diff
@@ -17,67 +21,101 @@ components:
...
+  plugins/node/opentelemetry-instrumentation-fastify: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:61]: Originally contributed by @obecny  

Would you care to take ownership?

```suggestion
  plugins/node/opentelemetry-instrumentation-fastify:
    - obecny
```

File: .github/component_owners.yml [hunk 12]
```diff
@@ -17,67 +21,101 @@ components:
...
   plugins/node/opentelemetry-instrumentation-generic-pool:
     - rauno56
   plugins/node/opentelemetry-instrumentation-graphql:
     - obecny
+  plugins/node/opentelemetry-instrumentation-hapi: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:67]: Originally contributed by @carolinee21 

Would you care to take ownership?

```suggestion
  plugins/node/opentelemetry-instrumentation-hapi:
    - carolinee21
```

File: .github/component_owners.yml [hunk 13]
```diff
@@ -17,67 +21,101 @@ components:
...
   plugins/node/opentelemetry-instrumentation-ioredis:
     - blumamir
     - naseemkullah
   plugins/node/opentelemetry-instrumentation-knex:
     - rauno56
-  plugins/node/opentelemetry-instrumentation-lru-memoizer:
-    - blumamir
+  plugins/node/opentelemetry-instrumentation-koa: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:74]: Originally contributed by @carolinee21 

Would you care to take ownership?

```suggestion
  plugins/node/opentelemetry-instrumentation-hapi:
    - carolinee21
```

File: .github/component_owners.yml [hunk 14]
```diff
@@ -17,67 +21,101 @@ components:
...
   plugins/node/opentelemetry-instrumentation-mysql:
     - haddasbronfman
+  plugins/node/opentelemetry-instrumentation-mysql2: []
+    # Unmaintained?
   plugins/node/opentelemetry-instrumentation-nestjs-core:
     - rauno56
-  plugins/node/opentelemetry-instrumentation-redis:
-    - blumamir
+  plugins/node/opentelemetry-instrumentation-net: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:86]: Originally contributed by @seemk 

Would you care to take ownership?

```suggestion
  plugins/node/opentelemetry-instrumentation-net:
    - seemk
```

File: .github/component_owners.yml [hunk 15]
```diff
@@ -17,67 +21,101 @@ components:
...
     - mhennoch
     - t2t2
   plugins/web/opentelemetry-instrumentation-user-interaction:
     - obecny
+  plugins/web/opentelemetry-plugin-react-load: []
+    # Unmaintained?
   propagators/opentelemetry-propagator-aws-xray:
     - carolabadeer
+  propagators/opentelemetry-propagator-grpc-census-binary: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:113]: Originally contributed by @nkelly75  

Would you care to take ownership?

```suggestion
  propagators/opentelemetry-propagator-grpc-census-binary: []
    - nkelly75
```

File: .github/component_owners.yml [hunk 16]
```diff
@@ -17,67 +21,101 @@ components:
...
   propagators/opentelemetry-propagator-instana:
     - basti1302
     - kirrg001
+  propagators/opentelemetry-propagator-ot-trace: []
+    # Unmaintained?
```
Comment (dyladan) [inline .github/component_owners.yml:118]: Originally contributed by @mwear  

Would you care to take ownership?

```suggestion
  propagators/opentelemetry-propagator-ot-trace:
    - mwear
```

File: .github/component_owners.yml (hunk 9 above)
Comment (Flarna) [reply .github/component_owners.yml:57]: I haven't created and I don't want to take ownership.
I did only the conversion from plugin to instrumentation.
Can't remember who initially developed it. Best guess is that @OlivierAlbertini did a port from opencensus in the early days.

File: .github/component_owners.yml [hunk 17]
```diff
@@ -17,67 +21,101 @@ components:
...
   plugins/node/opentelemetry-instrumentation-memcached:
     - rauno56
   plugins/node/opentelemetry-instrumentation-mongodb:
     - osherv
-  plugins/node/opentelemetry-instrumentation-mongoose:
-    - blumamir
   plugins/node/opentelemetry-instrumentation-mysql:
     - haddasbronfman
+  plugins/node/opentelemetry-instrumentation-mysql2: []
+    # Unmaintained?
```
Comment (pichlermarc) [inline .github/component_owners.yml:82]: @haddasbronfman agreed to take this one in a discussion we had on the CNCF Slack :slightly_smiling_face:

```suggestion
  plugins/node/opentelemetry-instrumentation-mysql2:
    - haddasbronfman
```

File: .github/component_owners.yml (hunk 14 above)
Comment (seemk) [reply .github/component_owners.yml:86]: Sure

File: .github/component_owners.yml (hunk 5 above)
Comment (seemk) [reply .github/component_owners.yml:51]: :+1:

**Comment by seemk** [comment]
I can take ownership of the instrumentations proposed to me. But what does it entail exactly?

File: .github/component_owners.yml (hunk 10 above)
Comment (vmarchaud) [reply .github/component_owners.yml:59]: I don't think that would be a good idea, folks at Aspeco have their own fork/version (https://github.com/aspecto-io/opentelemetry-ext-js/tree/master/packages/instrumentation-express) maybe @blumamir might be interested ?

**Comment by dyladan** [comment]
> I can take ownership of the instrumentations proposed to me. But what does it entail exactly?

It is mostly a commitment that you will respond to issues and prs in a timely manner. Maintainers can't pay attention to every package all the time so we rely on component owners to help us make decisions about their development

File: .github/component_owners.yml (hunk 3 above)
Comment (pichlermarc) [reply .github/component_owners.yml:25]: Yep, I agree, I'm not to familiar with the package, but it makes sense for us to take it. I'll get familiar with it in time. :+1:

File: .github/component_owners.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (pichlermarc) [reply .github/component_owners.yml:29]: Yep :+1:

File: .github/component_owners.yml (hunk 1 above)
Comment (dyladan) [reply .github/component_owners.yml:9]:
```suggestion
  detectors/node/opentelemetry-resource-detector-gcp:
    - aabmass
    - punya
```

**Review by pichlermarc — APPROVED** [review]

**Pull request merged**